internal/xds/translator/testdata/out/xds-ir/mixed-tls-jwt-authn.listeners.yaml

```diff
@@ -14,26 +15,6 @@
           initialStreamWindowSize: 65536
           maxConcurrentStreams: 100
         httpFilters:
-        - name: envoy.filters.http.jwt_authn
```

Comment:
why is this changing ?

Reply:
because this test file has been ignored (not explicit defined in test cases) until now, its content is outdated.